code/datums/uplink_items/uplink_general.dm

```diff
@@ -83,6 +83,9 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	var/refundable = FALSE
 	var/refund_path = null // Alternative path for refunds, in case the item purchased isn't what is actually refunded (ie: holoparasites).
 	var/refund_amount // specified refund amount in case there needs to be a TC penalty for refunds.
+	var/uses_special_spawn = FALSE // Our special little snowflakes that have to be spawned in a different way
+
+#define UPLINK_SPECIAL_SPAWNING "ONE PINK CHAINSAW PLEASE"
```

Comment:
Move this to the top of the file and #undef it at the bottom please

Reply:
I can't undef it as I'm using it in uplink_traitor.dm